packages/api/src/controllers/loginController.js

```diff
@@ -53,6 +47,24 @@ class loginController extends baseController {
       }
     };
   }
+
+  static loginWithGoogle() {
+    return async (req, res) => {
+      try {
+        const { sub: user_id, email, given_name: first_name, family_name: last_name } = req.user;
+        const user = await userModel.query().findById(user_id);
+        if (!user) {
```

Comment:
Wouldn't this check be something that should happen on Google's side?

Reply:
https://developers.google.com/identity/sign-in/web/sign-in